CHANGELOG.md

```diff
@@ -0,0 +1,2 @@
+# 3.0.0

```

Comment:
Looks like Releases has been used historically. Any reason not to use that?

Reply:
oh snap, yeah I can definitely use that. But that'll be gone after the monorepo merge?

Reply:
I prefer using GitHub Releases. I think we can make this file after migrated to monorepo If it's depends on monorepo.

Reply:
lerna also creates tags for each published package so we could continue to create releases for each package separately.